templates/activity.html

```diff
@@ -1,41 +1,39 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
-
-<head>
+  <head>
     <title>Community Activity</title>
-    <style>
-    canvas {
+      <style>
+        canvas {
         -moz-user-select: none;
         -webkit-user-select: none;
         -ms-user-select: none;
-    }
-    </style>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-</head>
-
-<body>
+        }
+      </style>
+      <meta charset="utf-8">
+      <meta charset="utf-8" content="text/html">
```

Comment:
This is probably not needed, 2 meta charset?

Reply:
I think I should remove the line-12 meta tag?